packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -86,8 +95,13 @@ contract Ethernauts is ERC721Enumerable, Ownable {
     // --------------------
 
     function mint() external payable onlyOnState(SaleState.Open) {
-        require(msg.value >= mintPrice, "Invalid msg.value");
-        require(availableToMint() > 0, "No available supply");
+        if (msg.value < mintPrice) {
+            revert CustomError("Invalid msg.value");
+        }
+
+        if (availableToMint() < 0 ) {
```

Comment:
This can just be `availableToMint() == 0` because availableToMint() cannot be below 0